src/pepr/operator/crd/sources/package/v1alpha1.ts

```diff
@@ -303,6 +303,19 @@ const sso = {
           type: "string",
         },
       },
+      groups: {
+        description: "The client sso group type",
+        type: "object",
+        properties: {
+          anyOf: {
+            description: "List of group allowed to access to client",
```

Comment:
```suggestion
            description: "List of groups allowed to access to client",
```

Also might be good to add the CRD gen task fix here so we're not carrying around that issue while modifying the CRD type, can steal from my draft PR - https://github.com/defenseunicorns/uds-core/pull/511/files#diff-76783f82b8fa78a37e73d203db3f8168d44cafb1b52fa7dc1df445e68198b7b7

Reply:
updated the typo and the gen crd task